pool_test.go

```diff
@@ -200,3 +200,13 @@ func ExampleGet() {
 	// length 100
 	// capacity 128
 }
+
+func TestBufferPoolGetWithNegativeValue(t *testing.T) {
+	for i := 0; i < 50; i++ {
+		b := Get(-20008899456)
```

Comment:
This is an int overflow on 32 bit systems. This should probably use a random value (or just `-i`?).

Reply:
I've changed it up, thanks @marten-seemann!